integration/tests/cook/test_basic.py

```diff
@@ -244,10 +245,11 @@ def test_failing_submit(self):
         self.assertEqual('failed', instance['status'], message)
```

Comment:
Which `instance` is this line referring to?

Reply:
This is a different test (`test_failing_submit`)

Reply:
Oops.